index.html

```diff
@@ -146,97 +163,99 @@ <h2 class="header tonic__header" >Multi-Post
         </button>
       </div>
     </div>
+  </div>
+  <!-- end of works -->
 
-    <div class="about">
-        <div class="landing about__landing">
-          <h2 class="header landing__header header__about">
-            <span>About</span>
-            <span>Myself</span>
-          </h2>
-            <p class="landing__text">
-              Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <div><a href="#" class="landing__link">
-            let's connect
-            </a></div>
-            <div class="landing__social-media">
-              <ul>
-                <li><a href="#">
-                  <object class="landing__icon" data="/img/github.svg" height="20" width="20"></object>
-                </a></li>
-                <li><a href="#">
-                  <object class="landing__icon" data="/img/Linkedin_icon.svg" height="20" width="20"></object>
-                </a></li>
-                <li><a href="#">
-                  <object class="landing__icon" data="/img/hands.svg" height="20" width="20"></object>
-                </a></li>
-                <li><a href="#">
-                <object  class="landing__icon" data="/img/twitter.svg" height="20" width="20"></object>
-                </a></li>
-                <li><a href="#">
-                  <object class="landing__icon" data="/img/medium.svg" height="20" width="20"></object>
-                </a></li>
-              </ul>
-          </div>
-          <button class="tonic__btn" type="button">
-          Get my resume
-          </button>
+  <!-- About Me -->
+  <div class="about" id="about-id">
```

Comment:
- [ ] [REQUIRED] It's better use `<section>` element instead of `<div>` element for semantic purposes ♻️.